eras/conway/impl/src/Cardano/Ledger/Conway/Rules/Ratify.hs

```diff
@@ -217,11 +213,13 @@ ratifyTransition = do
                   }
           trans @(ConwayRATIFY era) $ TRC (env, st', RatifySignal sigs)
         else do
+          -- This action hasn't been ratified yet. Process the remaining actions.
           st' <- trans @(ConwayRATIFY era) $ TRC (env, st, RatifySignal sigs)
-          if expired
-            then -- Action expired, do not include it in the next epoch
+          -- Finally, filter out actions that are not processed.
+          if gasExpiresAfter >= reCurrentEpoch
```

Comment:
For consistency with DRep expiration, the spec and the name of the record (After) :slightly_smiling_face: 
```suggestion
          if gasExpiresAfter > reCurrentEpoch
```

Reply:
:face_with_head_bandage: This mistake made me feel a bit sick :face_with_spiral_eyes:. Thanks for catching this!